src/ray/raylet/placement_group_resource_manager.h

```diff
@@ -100,6 +107,9 @@ class NewPlacementGroupResourceManager : public PlacementGroupResourceManager {
 
   bool PrepareBundle(const BundleSpecification &bundle_spec);
 
+  bool PrepareBundles(
+      const std::vector<std::shared_ptr<const BundleSpecification>> &bundle_specs);
+
```

Comment:
Can you move `PrepareBundle` method to the private method?

Reply:
updated